packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-list.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const BarriersList = ({
+    active_item_class_name,
+    base_class_name,
```

Comment:
```suggestion
    active_item_classname,
    base_classname,
```
@kate-deriv just a nit-pick)

Reply:
Done, thank you. I also change hover_item_class_name -> hover_item_classname